src/main/resources/templates/challenge.html

```diff
@@ -15,12 +15,185 @@
 		<a href="/dashboard" class="primary"><i class="fa fa-graduation-cap"></i>Submit</a>
 	</div>
 </div>
-<div class="challenge-tri">
-<div><textarea class="challenge-input" id="challenge-input">
-// Challenge: Type something that triggers the pretty print.
-// Try something like "var a = document.getElementById('hi');"
-</textarea></div>
-	<div><pre><code id="challenge-pretty" class="js"></code></pre></div>
+<div class="challenge-dual-pane">
+	<div class="left">
+		<div class="challenge-side-header">
+			<div class="title">Simple Dependency Injection</div>
+			<div class="subtitle">Java Challenge</div>
+		</div>
+		<div class="challenge-side-body">
+			<div class="block">
+				Implement a simple dependency injection library that has a single
+				<code>#provide</code> method. You may assume the following:
+			</div>
+			<ul class="block">
+				<li>All injectable constructors are annotated with <code>@Inject</code>.</li>
+				<li>A class named <code>InjectException</code> exists and extends <code>Exception</code>.</li>
+				<li>Only the core Java JDK is allowed. All versions are allowed.</li>
+			</ul>
+			<div class="block">
+				Any other non-listed assumptions must be handled appropriately.
+			</div>
+		</div>
+	</div>
+	<div class="right">
+		<div>
+			<textarea class="code-highlight" id="challenge-input">import java.lang.annotation.Annotation;
+import java.lang.reflect.AnnotatedElement;
```

Comment:
I didn't push to the server, but can you highlight the part that's underlined and screenshot it again? The way it's set up is a <textarea> is laying over the <pre><code> block and the styles between the two should match exactly. The textarea is providing the ability to edit, show a caret, and highlight (not color, think copy/paste type of highlight) text, and the pre/code block is there to provide the syntax highlighting. The underlining of what appears to be blank space may be caused by the two not aligning properly.